README.md

```diff
@@ -1 +1,38 @@
-# cr-training-mobile-menu
\ No newline at end of file
+# cr-training-mobile-menu
+
+> This is the skeleton and setup of mobile version for my portfolio
+
+![screenshot]
+Headline                                                                                 |
+:----------------------------------------------------------------------------------------|
+![](./screenshots.png)                                                   `               |
+
+
+## Built With
+
+- HTML & CSS & JAVASCRIPT
+
+## Link to project online
+- Coming soon...
+
```

Comment:
- [ ] [OPTIONAL] I think it would be nice to include a link to your project rather than putting a text that says **_' Coming Soon...'_**. Providing a link to the live demo of your project is a smart and fast way to show recruiters the actual implementation of your code in a live session.